recipes/lumen/LICENSE.txt

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2018-2020, HoloViz team (holoviz.org).
```

Comment:
The PyPI tarball already has `LICENSE` with the same content. Is there any need to include this here?

Reply:
No.